samples/Client/DaprClient/Readme.md

```diff
@@ -24,9 +24,28 @@ Deleted State!
 Executing transaction - save state and delete state
 Executed State Transaction!
 State not found in store
+Published deposit event!
 Done
  ```
 
+If you want to invoke RoutingService via HTTP or GrpcService via gRPC, you should run [RoutingService](..\..\AspNetCore\RoutingSample) or [GrpcService](..\..\AspNetCore\GrpcServiceSample) sample firstly, then run this command:
+
+``` sh
+ dapr run --app-id gRPC_Client -- dotnet run --useRouting true
+```
+
+or
+
+``` sh
+ dapr run --app-id gRPC_Client -- dotnet run --useGrpcsample true
+```
+
+or pass these 2 options to invoke RoutingService via HTTP and GrpcService via gRPC:
+
+``` sh
+ dapr run --app-id gRPC_Client -- dotnet run --useRouting true --useGrpcsample true
```

Comment:
I tried running the Routing and GrpcService sample together to try this command but I see this error:-
== APP == crit: Microsoft.AspNetCore.Server.Kestrel[0]

== APP ==       Unable to start Kestrel.

== APP == System.IO.IOException: Failed to bind to address http://127.0.0.1:5000: address already in use.

== APP ==  ---> Microsoft.AspNetCore.Connections.AddressInUseException: Address already in use

Need additional instructions..

Reply:
I changed GrpcServiceSample's port to 5050, so you can run both services simultaneously